core/jobs.py

```diff
@@ -850,8 +853,9 @@ def enqueue(cls, job_id, additional_job_params=None):
 
 
 class MultipleDatastoreEntitiesInputReader(input_readers.InputReader):
-    """Multiple Datastore Entities Input Reader, these are used to read values
-    from datastore and passed to mapper functions in MapReduce jobs.
+    """This is an Input Reader, these are used to read values from multiple
```

Comment:
This InputReader class is used to read values from ...

Reply:
done.